runtime/onert/core/src/dumper/dot/DotDumper.cc

```diff
@@ -142,8 +142,11 @@ void DotDumper::dump(const std::string &tag)
     for (auto output : op.getOutputs())
     {
       using onert::dumper::dot::Operand;
-      auto &output_node = operand_nodes.at(output);
-      node->addOutEdge(output_node.get());
+      if (!output.undefined())
```

Comment:
@wateret I wonder we need the same check for `input`.

Reply:
It looks like that case it is covered in https://github.com/Samsung/ONE/pull/4760/files/a9ef53c9fcf45029c01a98a573d109a67fc9d17b#diff-3637edfc73618817d72bc8de3c9044a8a00c79defcc336ded6deece4a824bfa1R135

Reply:
@wateret The link above seems to be this PR itself. Could you please explain what yo mean more?

Reply:
Oh, I meant line 135, in this PR. Sorry the link didn't work 😅

Reply:
The link works, but the collapsed source code doesn't seem to open automatically. :)

Reply:
@wateret 

1. At line 135, I see

```c
if (operand_nodes.find(input) == operand_nodes.end())
```

It seems `operand_nodes` is supposed not to have `undefined`. Is it right?

2. For your information, I've used a kind of `filter`-like syntax

```c
for (auto output : node.getOutputs() | ir::Remove::UNDEFINED)
```
to remove `undefined`.

Reply:
1. Yes
2. Thank looks better, I will fix!